lib/cache/fetchAndExtract.js

```diff
@@ -29,6 +29,7 @@ module.exports = async ({ url, name, tmpLoc, outLoc, raw, requestOpts }) => {
             reporter.debug(`[CACHE]   to ${chalk.bold(outLoc)}`)
             reporter.debug(`[CACHE]   tmpLoc ${chalk.bold(tmpLoc)}`)
             await mkdirp(tmpLoc)
```

Comment:
This (mkdirp outloc) should actually be outside the if block so that it works with both archives and regular files

Reply:
(picked the wrong line on my phone, it should be line 32)